templates_test/singleton/boil_suites_test.tpl

```diff
@@ -336,5 +336,4 @@ func TestUpsert(t *testing.T) {
   t.Run("{{$tableName}}", test{{$tableName}}Upsert)
   {{end -}}
   {{- end -}}
-}
-{{- end -}}
\ No newline at end of file
+}
```

Comment:
I notice whatever editor you're using is not putting newlines at the end of files. It would be a great favor if you could add that. Plays nice with git and other unix-like tools :) This is merely a friendly request and not a demand.